backends/events/default/default-events.cpp

```diff
@@ -160,11 +160,10 @@ bool DefaultEventManager::pollEvent(Common::Event &event) {
 		if (_vk->isDisplaying()) {
 			_vk->close(true);
 		} else {
+			PauseToken pt;
```

Comment:
You're not handling pt anywhere, so why are you introducing it as a new variable?